tools/contrib/junit_xml/__init__.py

```diff
@@ -0,0 +1,322 @@
+#!/usr/bin/env python
```

Comment:
Also, it looks like `test.py` shells out to `run_test.py`?  Why do we need both scripts?  If they are different in function, why not just call `run_test.py` instead of shelling out to it, so we don't have the complexity of multiple python environments?